README.md

```diff
@@ -28,6 +28,10 @@ The specific events currently being raised are
 *   **OFFENDER_REGISTRATION_DELETED** is raised when an offender's registration is deleted. sourceId of the event is source id of the registration.
 *   **OFFENDER_REGISTRATION_DEREGISTERED** is raised when an offender's registration is deregistered. sourceId of the event is source id of the deregistation.
 *   **OFFENDER_MANAGEMENT_TIER_CALCULATION_REQUIRED** is raised when an offender's management tier needs recalculating. sourceId of the event is source id of the MANAGEMENT_TIER_EVENT.
+*   **SENTENCE_ORDER_REQUIREMENT_CHANGED** is raised when an offender's sentence order requirement is changed. sourceId of the event is requirementId.
+*   **SENTENCE_CHANGED** is raised when an offender's sentence is changed. sourceId of the event is sentenceId.
+*   **OFFENDER_MERGED** is raised when an offender's merge history is changed. sourceId of the event is mergeHistoryId.
+*   **CONVICTION_CHANGED** is raised when an offender's conviction is changed. sourceId of the event is convictionId.
```

Comment:
I think you are missing the `OFFENDER_OGRS_ASSESSMENT_CHANGE ` event is the documentation which would be something like 
OFFENDER_OGRS_ASSESSMENT_CHANGE when an OGRS assessment is added or updated

Reply:
could maybe also describe what OGRS means?

Reply:
@dnyanesh-sonwane it stands for Offender Group Reconviction Scale and is a score to calculate the chances of reoffending

Reply:
done